src/packages/utils/src/things/json-rpc/json-rpc-quantity.ts

```diff
@@ -92,7 +92,7 @@ export class Quantity extends BaseJsonRpcType {
       const trimmedBuffer =
         firstNonZeroByte === 0
           ? this.bufferValue
-          : this.bufferValue.subarray(firstNonZeroByte, length);
```

Comment:
The second argument here is `end`, which should not be confused with the argument `byteLength`  passed to `readUIntBE()` on line :108 (_I_ confused the two, which is why this regression exists).